org/mozilla/jss/tests/all.pl

```diff
@@ -711,14 +709,14 @@ sub print_case_result {
 print "\n================= Test Results\n";
 print "JSSTEST_SUITE: $testpass / $testrun\n";
 my $rate = $testpass / $testrun * 100;
-printf "JSSTEST_RATE: %.0f %\n",$rate;
+printf "JSSTEST_RATE: %.0f %%\n",$rate;
 
 if ($testpass ne $testrun) {
     printf "Test Status: FAILURE\n";
-    system("false");
     printf "to test failed tests set the classpath and run the command(s)\n";
     outputEnv();
+    exit 1;
```

Comment:
I wasn't sure the purpose of using `system("false")` so I changed it to `exit 1`... Perhaps it wasn't used due to [portability reasons](https://perldoc.perl.org/perlport.html#exit) but those platforms aren't our main candidates.